src/main/java/org/terasology/scenario/internal/ui/LogicTree.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.scenario.internal.ui;
+
+import org.terasology.rendering.nui.widgets.treeView.Tree;
+
+public class LogicTree extends Tree<LogicTreeValue> {

```

Comment:
theoretically it might be possible / a good idea to combine LogicTree and LogicTreeValue into a single class. You just have to see how it turns out later for you. So no need to change it now but just keep in mind that it is an option that we might want to go for for simplicity